node/src/main/resources/defaults.conf

```diff
@@ -284,6 +284,12 @@ casper {
       047f0f0f5bbe1d6d1a8dac4d88a3957851940f39a57cd89d55fe25b536ab67e6d76fd3f365c83e5bfe11fe7117e549b1ae3dd39bfc867d1c725a4177692c4e7754
     ]
 
+    # Public key (default value) for genesis block
+    # Base16 encoding of the public key to use for signing a proposed blocks. Can be inferred from the private key for
+    # some signature algorithms
+    genesis-pos-vault-pub-key =
+      0432946f7f91f8f767d7c3d43674faf83586dffbd1b8f9278a5c72820dc20308836299f47575ff27f4a736b72e63d91c3cd853641861f64e08ee5f9204fc708df6
```

Comment:
Please remove _genesis_ from the name, it's already in genesis section - `pos-vault-public-key`.

Also `pos-multi-sig-public-keys` and `pos-multi-sig-quorum` belongs together so this new parameter should be below.

---

This public key is used to make transfer from PoS vault defined in PoS contract, not for signing blocks.